tests/__init__.py

```diff
@@ -424,6 +424,35 @@ def test_modified_uids(self):
                 ),
             )
 
+    def test_uids_mac_os(self):
+        """tests if we can build an image with a custom uid and gid of odoo"""
+        uids_dir = join(SCAFFOLDINGS_DIR, "uids_mac_os")
+        for sub_env in matrix():
+            self.compose_test(
+                uids_dir,
+                sub_env,
+                # verify that odoo user has the given ids
+                ("bash", "-c", 'test "$(id -u)" == "501"'),
+                ("bash", "-c", 'test "$(id -g)" == "20"'),
+                ("bash", "-c", 'test "$(id -u -n)" == "odoo"'),
+                # all those directories need to belong to odoo (user or group odoo/dialout)
+                (
+                    "bash",
+                    "-c",
+                    'test "$(stat -c \'%U:%G\' /var/lib/odoo)" == "odoo:dialout"',
```

Comment:
Please use group numeric output for the test. It avoids wondering where that `dialout` keyword comes from.

```suggestion
                    'test "$(stat -c \'%U:%g\' /var/lib/odoo)" == "odoo:20"',
```

Reply:
changed in 6bd6eeb